Tools/machines/perlmutter-nersc/perlmutter_cpu_warpx.profile.example

```diff
@@ -11,7 +11,7 @@ module load cmake/3.24.3
 module load cray-fftw/3.3.10.6
 
 # optional: for QED support with detailed tables
-export BOOST_ROOT=/global/common/software/spackecp/perlmutter/e4s-23.05/default/spack/opt/spack/linux-sles15-zen3/gcc-11.2.0/boost-1.82.0-ow5r5qrgslcwu33grygouajmuluzuzv3
+export BOOST_ROOT=/global/common/software/spackecp/perlmutter/e4s-23.08/default/spack/opt/spack/linux-sles15-zen3/gcc-12.3.0/boost-1.83.0-2duyuvmto7nxhdxbmtdzfqta2zd2e6gp
```

Comment:
According to NERSC, this would be the default one specified by Spack:
```suggestion
export BOOST_ROOT=/global/common/software/spackecp/perlmutter/e4s-23.08/default/spack/opt/spack/linux-sles15-zen3/gcc-12.3.0/boost-1.83.0-nxqk3hnci5g3wqv75wvsmuke3w74mzxi
```

Reply:
Apologies for the late reply. This version works.